src/AccessibilityInsights.SharedUx/Controls/TestTabs/AutomatedChecksControl.xaml

```diff
@@ -59,10 +59,18 @@
                 </Grid>
             </Grid>
             <Border Grid.Row="1" BorderBrush="{DynamicResource ResourceKey=TabBorderBrush}" BorderThickness="0,1,0,0" VerticalAlignment="Stretch" HorizontalAlignment="Stretch">
-                <local:AutomatedChecksCustomListControl KeyboardNavigation.TabNavigation="Once" DataGridAccessibleName="{x:Static Properties:Resources.lblTitleContent}"
-                          x:Name="listControl" HorizontalAlignment="Stretch" VerticalAlignment="Stretch" BorderThickness="0"
-                          AutomationProperties.AutomationId="{x:Static Properties:AutomationIDs.AutomatedChecksResultsListView}" Background="Transparent">
-                </local:AutomatedChecksCustomListControl>
+                <ScrollViewer HorizontalScrollBarVisibility ="Disabled" VerticalScrollBarVisibility="Auto">
```

Comment:
Maybe this is just part of the design, but it looks off to me that the two lists independently scroll horizontally but scroll vertically together--why not enable horizontal scroll here as well?
![screenshot showing the framework and normal failures each scrolling horizontally separately while both scroll vertically together](https://user-images.githubusercontent.com/4615491/172901918-60f38111-3c31-47ee-9321-e8f509a560f2.png)

Reply:
Similarly, now the horizontal scroll bar no longer sticks to the bottom of the app and instead moves around with the listview (before on left, after on right):
![screenshot of ai-win automated checks view showing horizontal scroll at the bottom of the page before and in the middle of the page under the listview on the right](https://user-images.githubusercontent.com/4615491/172908236-5009400d-21dc-4096-82b9-1c089480a14c.png)
Maybe we're ok with this change, just want to make sure.